src/tests/system/tests/test_ldap_extra_attrs.py

```diff
@@ -42,3 +48,198 @@ def test_ldap_extra_attrs__filled(client: Client, provider: GenericProvider, att
+    :setup:
+        1. Create new user "user1"
+        2. Add "number:telephonenumber" to ldap_user_extra_attrs
+        3. Start SSSD
+    :steps:
+        1. Run "getent passwd user1" to store user to cache
+        2. Run ldbsearch command
+    :expectedresults:
+        1. User is found
+        2. "number" is not in the output
+    :customerscenario: False
+    """
+    provider.user("user1").add()
+
+    client.sssd.domain["ldap_user_extra_attrs"] = "number:telephonenumber"
+
+    client.sssd.start()
+
+    result = client.tools.getent.passwd("user1")
+    assert result is not None, "User is not found"
+    assert result.name == "user1", "User has wrong name"
+
+    search = client.ldb.search(
+        f"/var/lib/sss/db/cache_{client.sssd.default_domain}.ldb", f"cn=users,cn={client.sssd.default_domain},cn=sysdb"
+    )
+
+    user_dict = search["name=user1@test,cn=users,cn=test,cn=sysdb"]
+    with pytest.raises(KeyError):
+        user_dict["number"]
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap_extra_attrs__extra_email(client: Client, ldap: LDAP):
+    """
+    :title: SSSD starts with ldap_user_email and ldap_user_extra_attrs and checks cached attributes
+    :setup:
+        1. Create new user "user1", set them mail and gecos
+        2. Edit config - ldap_user_extra_attrs = "email:mail, description:gecos" and ldap_user_email = "mail"
+        3. Start SSSD
+    :steps:
+        1. Run "getent passwd user1" to store user to cache
+        2. Run ldbsearch command to get cached info
+    :expectedresults:
+        1. User is found
+        2. "mail" and "email" are in the output with correct value
+    :customerscenario: False
+    """
+    ldap.user("user1").add(gecos="gecos1", mail="user1@example.test")
+
+    client.sssd.domain["ldap_user_email"] = "mail"
+    client.sssd.domain["ldap_user_extra_attrs"] = "email:mail, description:gecos"
+    client.sssd.sssd["services"] = "nss, pam, ifp"
+    client.sssd.start()
+
+    result = client.tools.getent.passwd("user1")
+    assert result is not None, "User is not found"
+    assert result.name == "user1", "User has wrong name"
+
+    search = client.ldb.search(
+        f"/var/lib/sss/db/cache_{client.sssd.default_domain}.ldb", f"cn=users,cn={client.sssd.default_domain},cn=sysdb"
+    )
+
+    user_dict = search["name=user1@test,cn=users,cn=test,cn=sysdb"]
+    assert user_dict["description"] == ["gecos1"], "attribute 'descripion' was not correct"
+    assert user_dict["mail"] == ["user1@example.test"], "attribute 'mail' was not correct"
+    assert user_dict["email"] == ["user1@example.test"], "attribute 'email' was not correct"
+
+
+@pytest.mark.ticket(bz=1667252)
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_ldap_extra_attrs__ifp(client: Client, provider: GenericProvider):
+    """
+    :title: ifp do not crash when requesting extra attributes
+    :setup:
+        1. Create new user "user1"
+        2. Configure properly sssd.conf
```

Comment:
This step 2 comment doesn't provide any value to the reader, how about `configure 'test' ldap user extra attribute`